build/thirdparty/linux/Dockerfile

```diff
@@ -55,7 +55,7 @@ WORKDIR /usr/local
 RUN tar -czvf /kubebuilder_${OS}_${ARCH}.tar.gz kubebuilder/
 
 # Copy tarball to final image.
-FROM alpine:3.8
+FROM alpine:3.16
```

Comment:
👍

Reply:
Could we have a PR to update only one thing per time?
Why? After we get merged the envtools will be re-build and we can check if all work fine. 
Otherwise, we need to revert the changes.

Reply:
Sure. They can be split.